### PR TITLE
Add eventsTracker staging url

### PR DESCRIPTION
**Pull request by jrlarano**
Relates to: https://github.com/tjek/octopus/issues/561

### CODE REVIEW

File: lib/sgn-sdk.ts [hunk 1]
```diff
@@ -67,7 +67,13 @@ if (isBrowser()) {
 
         if (trackId) scriptConfig.eventTracker = new Tracker({trackId});
 
-        if (environment === 'staging') scriptConfig.coreUrl = coreUrlStaging;
+        if (environment === 'staging') {
```
Comment (mortenbo) [inline lib/sgn-sdk.ts]: IMO we shouldn't make two trackers in a row like this.

File: lib/sgn-sdk.ts [hunk 2]
```diff
@@ -67,7 +67,13 @@ if (isBrowser()) {
 
         if (trackId) scriptConfig.eventTracker = new Tracker({trackId});
```
Comment (klarstrup) [inline lib/sgn-sdk.ts]: You can also just skip creating an eventTracker entirely at this step, if the environment is staging.

**Review by mortenbo — APPROVED** [review]
Can you/we make sure to always use curly brackets around conditions etc.?

File: lib/sgn-sdk.ts [hunk 3]
```diff
@@ -65,9 +65,10 @@ if (isBrowser()) {
 
         if (apiKey) scriptConfig.apiKey = apiKey;
 
-        if (trackId) scriptConfig.eventTracker = new Tracker({trackId});
+        if (trackId && !isStaging)
+            scriptConfig.eventTracker = new Tracker({trackId});
 
-        if (environment === 'staging') scriptConfig.coreUrl = coreUrlStaging;
+        if (isStaging) scriptConfig.coreUrl = coreUrlStaging;
```
Comment (mortenbo) [inline lib/sgn-sdk.ts]: I would move this above new Tracker and after if apiKey. Weird to set it after new Tracker IMO

**Comment by jrlarano** [comment]
Hi @klarstrup, need your help in deploying the develop branch 🙂

**Comment by klarstrup** [comment]
Live now on all channels https://github.com/tjek/tjek-js-sdk/releases/tag/release-18.09.2024-14.07

**Pull request merged**